pro/src/routes/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './Profile'
```

Comment:
```suggestion
export { default as Profile } from './Profile'
```
J'aurai peut etre préfixé en `UserProfile`, afin qu'on ai toute les route User ensemble.
Dans l'idée ca veux dire que lorsqu'on fera les routes Login, logout, signup, etc on utilisera UserLogin, UserSignUp, etc